python/paddle/utils/plot.py

```diff
@@ -50,13 +54,21 @@ def __plot_is_disabled__(self):
         return self.__disable_plot__ == "True"
 
     def append(self, title, step, value):
+        '''Feed data
+	:param title: the title of the figure
+	:param step: x_axis
+	:param value: y_axis
+	'''
```

Comment:
please follow the code style
https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/parallel_executor.py#L33

Reply:
> please follow the code style
> https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/parallel_executor.py#L33

Sure, I am already add the usage now~

Reply:
> please follow the code style
> https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/parallel_executor.py#L33

Sure~